robosherlock/descriptors/analysis_engines/demo.yaml

```diff
@@ -9,5 +9,8 @@ fixedflow:
   - ImageSegmentationAnnotator
```

Comment:
change this file back. Committing these changes breaks tutorials.

Reply:
done